veloroutes_voies_vertes/processing/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
I think in your python editor, you need to create a "Python file", not a Python script. it might be the reason why your IDEA is adding these lines. Like in #1